README.md

```diff
@@ -1,28 +1,76 @@
-# README
-
 # Classroom Dashboard - KBOP
--This is the KBOP team version of Classroom Dashboard, a classroom engagement tool written for Harvey Mudd College's CS121
 
-* Ruby version: 2.5.0
+Classroom Dashboard is a web-based, ruby on rails platform to help professors manage in-classroom interaction. The dashboard complements existing lecture methods, giving professors a suite of tools to help them better engage their classes. These tools include equity cards to randomly call on students, polls for real-time feedback, and image assignments for participation. Classroom Dashboard takes advantage of all the technology already in classrooms to turn distractions into educational assets.
+
+## Architecture
+
+### Prerequisites
+
+1. Git
+2. Ruby 2.5.0 or later
+3. Rails 5.0 or later
+4. ImageMagick libraries (can be acquired either by downloading the binaries or using downloaders like MacPorts or Apt-Get)
+
+### Gems
+
+1. omniauth, for Google OAuth integration
+2. chartkick, for JS chart generation
+3. bootstrap-sass, for frontend templating
+4. jquery-rails, for frontend JS libraries
+5. paperclip, for image management
+
+## Installation
+
+1. Clone the repository
+```shell
```

Comment:
Should this look different?

Reply:
I think this is how it's supposed to look, but I'm not positive